src/components/Dropdown/props.ts

```diff
@@ -80,6 +80,15 @@ export type DropdownClickEvent = {
   selectedPath: string[],
 }
 
+export enum OpenChangeInfoSource {
+  Trigger='trigger',
```

Comment:
please use consistent spacing with other enums

```suggestion
  Trigger = 'trigger',
```

Reply:
fixed